base_station/gui/src/components/CameraSelection.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="wrap2">
-    <span v-if="cam_index >= 0" class="title">Current Camera: {{cam_index}}</span>
+    <span v-if="cam_index >= 0" class="title">Current Camera: {{cam_index - 1}}</span>
```

Comment:
change to cam_index. (instead redefine what cam_index is by subtracting it in its definition